src/frontend/components/SelectContent/index.spec.tsx

```diff
@@ -143,6 +151,24 @@ describe('<SelectContent />', () => {
                   is_ready_to_show: true,
                 }),
               ]}
+              webinars={[
```

Comment:
I'm worried that this might be hardcoded when we are going to add many other ressources to marsha. Am I right?

Reply:
Here are only 'core' resources.
Added ones are on their dedicated app.

Reply:
Ok thank you for the explaination! So when we will create the `interactive` app separated from the `core` app, all resources will be in their own app :+1: